### ISSUE TITLE
Update does not skolemize the subject

### ISSUE THREAD

**Issue by danielbeeke** [LDflex/LDflex#262]
See https://github.com/comunica/comunica/issues/985

**Comment by jeswr** [comment]
A PR with your suggested fix is welcome!

**Comment by danielbeeke** [comment]
Yes, one thing though, I found that when I add skolemize it would not accept the following:

```
await path.address.delete(streetPath)
```

If streetPath needs skolemization it will not automatically resolve.
Ill make a test for this case and submit a PR.

**Comment by danielbeeke** [comment]
I checked the tests, at the moment it is not a supported feature to resolve paths and then delete them.
